setup-dev-env.sh

```diff
@@ -27,9 +27,11 @@ while [ "$1" != "" ]; do
         # Disable installation of 'cuda-drivers' in the role 'cuda'.
         option_no_cuda_drivers=true
         ;;
-    --no-dev)
-        # Disable installation dev packages .
-        option_no_dev=true
+    --runtime)
```

Comment:
Actually, it's not `perf`. It should be done in another PR as `feat`.

Reply:
Sure, I will keep it as it is in this PR.